src/Sylius/Behat/Page/Admin/Product/Common/ProductTaxonomyTrait.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Page\Admin\Product\Common;
+
+use Sylius\Behat\Service\AutocompleteHelper;
+use Sylius\Component\Taxonomy\Model\TaxonInterface;
+
+trait ProductTaxonomyTrait
+{
+    public function getDefinedProductTaxonomyElements(): array
+    {
+        return [
+            'main_taxon' => '#sylius_product_mainTaxon',
```

Comment:
Is it working? Should not be here?:
```suggestion
            'main_taxon' => '#sylius_admin_product_mainTaxon',
```

Reply:
https://github.com/Sylius/Sylius/pull/16349/files#diff-37ff1dc9ea94289d8f49e3199e5c2dd486e8e72d498cd9de71c114cd803b861eR110

Actually, the scenarios concerning product main taxon aren't enabled yet.